service-base/src/main/java/org/eclipse/hono/service/credentials/CredentialsService.java

```diff
@@ -65,11 +66,11 @@
      * @see <a href="https://www.eclipse.org/hono/api/credentials-api/#get-credentials">
      *      Credentials API - Get Credentials</a>
      */
-    void get(String tenantId, String type, String authId, Handler<AsyncResult<CredentialsResult<JsonObject>>> resultHandler);
+    void get(String tenantId, String type, String authId, JsonObject clientContext, Handler<AsyncResult<CredentialsResult<JsonObject>>> resultHandler);
```

Comment:
I would rather have an overloaded version of the *get* method instead of requiring client code to always provide `null` as a parameter when no client context is available. You can already see the (ugly) result in the test cases that you had to adapt accordingly...

Reply:
+1 I missed to add it. It's now there and I removed all ugly calls from the tests